pkgs/tools/text/sgrep/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, runCommand, lib, m4, makeWrapper }:
+let
+  sgrep = stdenv.mkDerivation rec {
+    pname = "sgrep";
+    version = "1.94a";
+
+    src = fetchurl {
+      url =
+        "https://www.cs.helsinki.fi/pub/Software/Local/Sgrep/sgrep-${version}.tar.gz";
```

Comment:
```suggestion
      url = "https://www.cs.helsinki.fi/pub/Software/Local/Sgrep/sgrep-${version}.tar.gz";
```

Reply:
Applied (editor was configured with nixfmt, not nixpkgs-fmt).